index.d.ts

```diff
@@ -16,5 +16,4 @@ declare class FastifyOtelInstrumentation<Config extends FastifyOtelInstrumentati
   plugin (): (instance: FastifyInstance, opts: FastifyOtelOptions, done: (err?: Error) => void) => void
 }
 
-export default FastifyOtelInstrumentation
-export { FastifyOtelInstrumentation }
+export = FastifyOtelInstrumentation
```

Comment:
Good spot!

Is this enough? Should we export the default as other plugins do?

https://github.com/fastify/fastify-redis/blob/713d5019c5bcc7003a0cf42c13cae56551074332/types/index.d.ts#L37-L41

Reply:
I don't recall spotting other issues on other plugins, but also I import them into register in an fp so maybe I missed the issues. If it's an issue here, then it's also presumably an issue elsewhere.

Reply:
Should we add some tests using eg tsd?

Reply:
Yes

Reply:
I mucked around with the tests, and I simply am not familiar enough with TSD or CJS/ESM nuance tests to know how to actually test for this.